lib/segment/src/entry/entry_point.rs

```diff
@@ -86,7 +86,6 @@ pub trait SegmentEntry: PartialSnapshotEntry {
         op_num: SeqNumberType,
         point_id: PointIdType,
         vector_name: &VectorName,
-        hw_counter: &HardwareCounterCell,
```

Comment:
why is this not measured anymore?

Reply:
We decided to not measure vector deletions